packages/utils/src/pick.ts

```diff
@@ -1,9 +1,14 @@
 /**
- * Pick keys from object.
+ * The `pick` function creates a new object by selecting specified keys from
+ * the original object. It takes an object and an array of keys to include,
+ * returning a new object that contains only the properties specified in the
```

Comment:
**thought:** should we wrap parameter names in template strings in the description aswell?